include/libopencm3/nrf/common/i2c.h

```diff
@@ -75,9 +75,9 @@
 /* Register Contents */
 
 /** @addtogroup i2c_shorts I2C event -> task shortcuts
- * Shortcut, which is activated causes that if certain event
- * occurrs, chosen task is automatically acitvated by the
- * hardware.
+ * The effect of activated shortcut is, that upon I2C event
```

Comment:
The term "shortcut" itself is used in reference manual, so this wording should be familiar to the NRF users.

Reply:
could we just use "shortcut" then instead of "shorts" ?